src/YEthStakerStrategy.sol

```diff
@@ -302,6 +331,8 @@ contract YEthStakerStrategy is
 
         // revoke previous allowance
         if (previous != address(0)) {
+            // review: don't you need safeApprove?
```

Comment:
SafeApprove is deprecated: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/dc44c9f1a4c3b10af99492eed84f83ed244203f6/contracts/token/ERC20/utils/SafeERC20.sol#L39

And it is removed from OZ 5.0 in favour of increase/decrease allowance: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/932fddf69a699a9a80fd2396fd1a2ab91cdda123/contracts/token/ERC20/utils/SafeERC20.sol#L61

Here we know we are dealing with the tokens that don't have a problem with allowance, like USDT. Do we have any benefit from using wrapper functions?